plugins/adr/src/components/AdrReader/AdrReader.tsx

```diff
@@ -44,12 +44,11 @@ export const AdrReader = (props: {
   const { entity } = useEntity();
   const scmIntegrations = useApi(scmIntegrationsApiRef);
   const adrApi = useApi(adrApiRef);
-  const adrLocationUrl = getAdrLocationUrl(entity, scmIntegrations);
+  const adrLocationUrl = getAdrLocationUrl(entity, scmIntegrations, adr);
 
-  const url = `${adrLocationUrl.replace(/\/$/, '')}/${adr}`;
   const { value, loading, error } = useAsync(
-    async () => adrApi.readAdr(url),
-    [url],
+    async () => adrApi.readAdr(adrLocationUrl),
+    [adrLocationUrl],
```

Comment:
You would want to create a second variable for the file path url (eg. `adrFileLocationUrl`) to use here since the original `adrLocationUrl` variable is used below as the baseUrl for the ADR decorators (line 66)